src/electionguard/serialize.hpp

```diff
@@ -585,7 +585,7 @@ namespace electionguard
             static string toJson(const electionguard::Manifest &serializable)
             {
                 auto manifestStr = fromObject(serializable).dump();
-                // special characters like ú are converted to ascii escapes in C# but then 
+                // special characters like ú are converted to ascii escapes in C# but then
                 //      https://github.com/nlohmann/json has a bug where it duplicates slashes,
                 //      so this code removes the duplicates to work around the bug
                 return regex_replace(manifestStr, regex("\\\\\\\\u"), "\\u");
```

Comment:
Added an issue to look at this line in the future.  Because of UTF-8 changes on the C# side, this might not be needed.